src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -2688,6 +2738,15 @@ internal static extern
             [In] IntPtr lpPassword
         );
 
+
+        [DllImport(PinvokeDllNames.SetServiceObjectSecurityDllName, CharSet = CharSet.Unicode, SetLastError = true)]
+        internal static extern
+        bool SetServiceObjectSecurity(
```

Comment:
@iSazonov This is supported, you can see in the documentation here:
https://docs.microsoft.com/en-us/windows/desktop/api/winsvc/nf-winsvc-setserviceobjectsecurity
![image](https://user-images.githubusercontent.com/10873629/55650533-ce588a80-579a-11e9-920e-f871e2c362d6.png)
Unfortunately, the dll names are only in the source of the HTML:
![image](https://user-images.githubusercontent.com/10873629/55650566-e9c39580-579a-11e9-8f91-fea40f173404.png)

Reply:
@TravisEz13 Thanks for confirmation! (We already caught problems with Win7, Core and others)